core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -15,11 +16,27 @@ final case class PersistentDisk(id: DiskId,
                                 size: DiskSize,
                                 diskType: DiskType,
                                 blockSize: BlockSize,
-                                labels: LabelMap)
+                                labels: LabelMap) {
+  def projectNameString: String = s"${googleProject.value}/${name.value}"
+}
 
 final case class DiskId(id: Long) extends AnyVal
 final case class DiskSamResourceId(asString: String) extends AnyVal
 
+/** Default persistent disk labels */
+case class DefaultDiskLabels(diskName: DiskName,
+                             googleProject: GoogleProject,
+                             creator: WorkbenchEmail,
+                             serviceAccount: WorkbenchEmail) {
```

Comment:
Not sure `serviceAccount` really makes sense as a disk label.. you don't need a service account to create a disk.

Reply:
what's the purpose of these labels? are they get created as google labels?

Reply:
@rtitle thanks - meant to leave a comment asking if SAs were relevant for disks. 
@Qi77Qi the disk labels don't get persisted to google (similar to runtimes) - it's just another way to filter disks like runtimes

Reply:
in that case...do we still need labels like `diskName, googleProject, creator`? We can filter disks by these already without labels I think?

Reply:
Still think it's useful even if you can filter by googleProject, diskName in other ways. For instance Terra UI lists runtimes like this:
```
https://notebooks.firecloud.org/api/google/v1/runtimes?saturnAutoCreated=true&googleProject=broad-firecloud-dsde&creator=rtitle%40broadinstitute.org
```
They could change to this: 
```
https://notebooks.firecloud.org/api/google/v1/runtimes/broad-firecloud-dsde?saturnAutoCreated=true&creator=rtitle%40broadinstitute.org
```
but either way the DB query is pretty fast. It's nice to support both options.